arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -0,0 +1,598 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2016-09-30"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/containerServices": {
+      "get": {
+        "tags": [
+          "ContainerServices"
+        ],
+        "operationId": "ContainerServices_List",
+        "description": "Lists all container services in a subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerServiceListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
@brendandburns - This should be pageable too `"nextLinkName": "nextLink"`
1. That would be consistent with the other list operation and technically a customer could have container services in a resource group that are more than what a server can return in 1 server page size.
2. The model returned by the operation "ContainerServiceListResult" does have a "nextLink"property defined in it. So  it makes more sense to have `"x-ms-pageable": { "nextLinkName": "nextLink" }`